man/.gitignore

```diff
@@ -1,7 +1 @@
-*.gz
```

Comment:
Yep. Many of the PRs I made during the holidays conflict with each other. I didn't want to wait for each PR to get merged before working on the next one. Though in this case, I could have just not touched the file, but I didn't think that far.

I will rebase once we get a merge conflict because another PR is merged.